spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -86,7 +86,12 @@ public ModuleRegistry moduleRegistry() {
 	protected static class LocalConfig {
 
 		@Bean
-		public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
+		public ModuleDeployer longRunningModuleDeployer(ModuleLauncher moduleLauncher) {
```

Comment:
-1 for using "stream and "task".  Those names imply that there is a context to what a short term process (task in diego) and an long term process (lrp in diego) are used, which really shouldn't be the case.  I realize that in this initial cut of the functionality that these deployers are only used in that case, but we shouldn't assume they always will be.